### ISSUE TITLE
Welcome Page: When enabled, space/enter key don't work

### ISSUE THREAD

**Issue by saibing** [onivim/oni#1459]
Oni: 

version: 0.2.22 
os: Windows 7

I try input space on anywhere such as command line, text editor etc. there is no any response for me.

![image](https://user-images.githubusercontent.com/5676390/35901405-647003c6-0c11-11e8-9608-8e9bf0fdafc4.png)
![image](https://user-images.githubusercontent.com/5676390/35901453-97f0d798-0c11-11e8-8c3b-0a7284a2642d.png)

**Comment by TalAmuyal** [comment]
Hi @saibing!

Do you have `"experimental.wildmenu.mode": true,` in your configuration file? is adding it solves the problem?
By `0.2.22`, do you mean `master`? if so, what commit?
Also, is it a regression for that version or are you trying it for the first time?

If all of the above didn't help, could you please explain what did you try to achieve and how? (I'm not sure I understand the problem)

**Comment by saibing** [comment]
- my config.js settings
![image](https://user-images.githubusercontent.com/5676390/35916133-5b238f6c-0c44-11e8-9b32-e59f71d7797d.png)

- oni version
![image](https://user-images.githubusercontent.com/5676390/35916277-d11d168e-0c44-11e8-9724-8b959067ccb2.png)

- my problem
![image](https://user-images.githubusercontent.com/5676390/35917340-b1047140-0c48-11e8-8e82-591c4db85d10.png)

I can not input space between 'set' with 'nolist'

**Comment by TalAmuyal** [comment]
@bryphe, do you know how to approach such an issue? might it be related to one of the latest input-handling PRs?

**Comment by bryphe** [comment]
@TalAmuyal - it's possible. I think it would be helpful for us to add logging in the `InputManager` too - we could get detailed tracing of which commands are passing the filter. My guess is that some command is swallowing the `space`/`enter` keys.

@saibing - do you have any custom keybindings in the `activate` section of your `config.js`?

**Comment by saibing** [comment]
- my config.js
```js
// For more information on customizing Oni,
// check out our wiki page:
// https://github.com/onivim/oni/wiki/Configuration

const activate = (oni) => {
    console.log("config activated")

    // Input 
    //
    // Add input bindings here:
    //
    //oni.input.bind("<c-enter>", () => console.log("Control+Enter was pressed"))
    oni.input.bind("<s-c-r>", "language.findAllReferences")
    oni.input.bind("<s-insert>", "editor.clipboard.paste")

    //
    // Or remove the default bindings here by uncommenting the below line:
    //
    // oni.input.unbind("<c-p>")
}

const deactivate = () => {
    console.log("config deactivated")
}

module.exports = {
    activate,
    deactivate,
    //add custom config here, such as

    //"ui.colorscheme": "solarized8_dark",
    "editor.fontSize": "15px",
    "editor.fontFamily": "Source Code Pro",

    "experimental.editor.textMateHighlighting.enabled": true,

    "experimental.markdownPreview.enabled": true,
    // language server configuration
    "language.go.languageServer.rootFiles": [".git"],
    //"language.go.completionTriggerCharacters": ["."],
    "language.go.languageServer.arguments": ["--gocodecompletion", "--freeosmemory", "false"],
   

   //"oni.useDefaultConfig": true,
   //"oni.bookmarks": ["~/Documents",]
   //"oni.loadInitVim": true,
   //"editor.fontSize": "14px",
   //"editor.fontFamily": "Monaco"

   // UI customizations
    "ui.animations.enabled": true,
    "ui.fontSmoothing": "auto",
    "experimental.browser.enabled": true,
    "experimental.commandline.mode": true,
    "experimental.commandline.icons": true,
    "experimental.wildmenu.mode": true,
    "experimental.welcome.enabled": true,

    "experimental.neovim.transport": "stdio"
    // TODO: Enable pipe transport for Windows
    // "experimental.neovim.transport": Platform.isWindows() ? "pipe" : "stdio",
}
```

I found that there is no problem if I use default config.js file.

**Comment by saibing** [comment]
@bryphe @TalAmuyal 
I found the problem occur if I enable the welcome page on startup

```js
"experimental.welcome.enabled": true,
```

there are some errors on developer tools:

```js
D:\work\git\ts\oni\node_modules\fbjs\lib\warning.js:33 Warning: Each child in an array or iterator should have a unique "key" prop.

Check the render method of `p`. See https://fb.me/react-warning-keys for more information.
    in s (created by p)
    in p (created by S)
    in S (created by Connect(S))
    in Connect(S) (created by b)
    in l (created by b)
    in div (created by styled.div)
    in styled.div (created by b)
    in div (created by styled.div)
    in styled.div (created by b)
    in b (created by Connect(b))
    in Connect(b) (created by i)
    in l (created by i)
    in div (created by i)
    in div (created by i)
    in i (created by i)
    in div (created by i)
    in div (created by i)
    in i (created by s)
    in div (created by s)
    in div (created by s)
    in s (created by _)
    in div (created by _)
    in div (created by _)
    in div (created by _)
    in div (created by _)
    in div (created by _)
    in ThemeProvider (created by _)
    in _ (created by Connect(_))
    in Connect(_)
    in l
/D:/work/git/ts/oni/lib/browser/bundle.js:1 Error: ENOENT: no such file or directory, stat 'D:\work\git\isource\ci\WELCOME'
/D:/work/git/ts/oni/lib/browser/bundle.js:1 Unhandled command: update_sp
t.warn @ /D:/work/git/ts/oni/lib/browser/bundle.js:1
/D:/work/git/ts/oni/lib/browser/bundle.js:1 Unhandled command: mode_info_set

```

**Comment by bryphe** [comment]
Thanks @saibing ! Glad you found the issue, appreciate the update (and that you're testing all the experimental features 👍 ). Seems like there are still some issues with the welcome page before we can enable it by default.

**Comment by bryphe** [comment]
Hope you don't mind - I updated the title to reflect the welcome page issue.

**Comment by saibing** [comment]
@bryphe 
It's does not matter

**Comment by Akin909** [comment]
should be fixed by #2552
